sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCSuite.scala

```diff
@@ -464,5 +476,6 @@ class JDBCSuite extends SparkFunSuite with BeforeAndAfter with SharedSQLContext
     assert(Postgres.getTableExistsQuery(table) == limitQuery)
     assert(db2.getTableExistsQuery(table) == defaultQuery)
     assert(h2.getTableExistsQuery(table) == defaultQuery)
+    assert(derby.getTableExistsQuery(table) == defaultQuery)
```

Comment:
Thanks for the review comments, Reynold and Michael. Hopefully, my last commit (e6022ed) addresses your concerns:

o I adjusted the indentation of a method signature, per Reynold's advice.

o I simplified an "if" expression, also per Reynold's advice. However, I have a question about this change (see my comment above on the line in question).

Thanks,
-Rick